db/migrate/20220411175218_create_tasks.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTasks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :tasks do |t|
+      t.citext :title, null: false, index: { unique: true }
+      t.citext :slug, null: false, index: { unique: true }
+      t.text :description
+      t.references :challenge, null: false, foreign_key: true
```

Comment:
+ index

Reply:
we have the index, it was added automatically. See here line 50 - https://github.com/DevChallengeIT/devchallenge-platform/blob/81e85721125592167133b5b939a6138f74cd7af0/db/schema.rb#L38-L54